src/sync/branches.ts

```diff
@@ -3,7 +3,9 @@ import { GitHubAPI } from "probot";
 import { Repository } from "models/subscription";
 import { GitHubInstallationClient } from "../github/client/github-installation-client";
 import Logger from "bunyan";
-import { numberFlag, NumberFlags } from "config/feature-flags";
+import { BackfillMessagePayload } from "~/src/sqs/backfill";
+import { NumberFlags } from "config/feature-flags";
+import { getCommitSinceDate } from "~/src/sync/sync-utils";
```

Comment:
/nit
Is `~/src/blah` a pattern we prefer? Or team prefer `./blah` (or even `../../blah` if is another folder?)

Reply:
if it's in the same folder, use `./blah`, if it's in a shortcut folder (like models) use `models/blah`, for anything else, use `~/src/some-dir/blah`